docs/connectors/io-crd-config/sink-crd-config.md

```diff
@@ -70,13 +70,20 @@ To use a secret in an environment variable in a Pod, follow these steps.
 2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
 3. Modify your image and or command line so that the program looks for values in the specified environment variables.
 
+Pulsar clusters supports using TLS or other authentication plugin for authentication.
+
+| Field | Description |
+| --- | --- |The authentication secret contains the `clientAuthenticationPlugin` and `clientAuthenticationParametersPath` fields.|
+| `tlsSecret` | The TLS secret contains the `use_tls`, `tls_allow_insecure`, `hostname_verification_enabled`, `tls_trust_cert_path` fields.|
+
 ## Packages
 
 Function Mesh supports running Pulsar connectors in Java.
 
 | Field | Description |
 | --- | --- |
 | `jarLocation` | Path to the JAR file for the connector.|
+| `extraDependenciesDir` | It specifies one or more dependent directories for the JAR package. |
```

Comment:
only one dir can be specified.

multiple lib/jar can be in that dir.

Reply:
Thanks for your comments. PTAL. thanks.